editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/internal/l10n/messages.properties

```diff
@@ -215,3 +215,20 @@ selectEndpointWizardWindowTitle=Select component
 updatePomDependenciesProgressDialogLabel=Updating Maven project file for new connector...
 loadingCamelFile=Loading {0} of project {1}
 pending=Pending ...
+
+# Preferred Labels
+preferredLabelsNewDialogTitle=New Preferred Label
+preferredLabelsEditDialogTitle=Edit Preferred Label
+preferredLabelsTitle=Select your preferred label
+preferredLabelsComponent=Component: 
+preferredLabelsComponentHeader=Component 
+preferredLabelsParameter=Parameter:  
+preferredLabelsParameterHeader=Parameter
+preferredLabelsErrorMessage=Input is not valid
```

Comment:
Maybe worth precising why it is not valid

Reply:
This message is not used, so I will remove it.